blackbox/stratisd.spec

```diff
@@ -49,6 +49,7 @@ a2x -f manpage docs/stratisd.txt
 # Daemon should be really private
 mkdir -p %{buildroot}%{_libexecdir}
 mv %{buildroot}%{_bindir}/stratisd %{buildroot}%{_libexecdir}/stratisd
+mv %{buildroot}%{_bindir}/stratis_uuids_to_names %{buildroot}%{_libexecdir}/stratis_uuids_to_names
```

Comment:
I think we need to find out how to specify a different directory. In https://github.com/stratis-storage/stratisd/pull/2101/files#diff-51690af9f42bd70c98547a192a58bcdaR1 the path to the script is ```/usr/lib/udev/stratis_uuids_to_names``` and we've been assured that that's the correct place for a udev rule to go.